python/chronos/script/benchmark_tool1.0.py

```diff
@@ -0,0 +1,338 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+import psutil
+import platform
+import subprocess
+import os
+import sys
+import logging
+import warnings
+from typing import Union, Dict, List, Optional
+import numpy as np
+import re
+
+from bigdl.nano.common.cpu_schedule import schedule_workers
+
+
+def get_bytesize(bytes):
+    """
+    Scale bytes to its proper format ( B / KB / MB / GB / TB / PB )
+    """
+    factor = 1024
+    for unit in ["B","KB","MB","GB","TB","PB"]:
+        if bytes < factor:
+            return str(format(bytes,'.2f')) + unit
+        bytes /= factor
+
+
+def CPU_info():
+    """
+    Capture hardware information, such as CPU model, CPU informations, memory status
+    """
+
+    #information about CPU
+    socket_num = int(subprocess.getoutput('cat /proc/cpuinfo | grep "physical id" | sort -u | wc -l'))
+    model_name = subprocess.getoutput('lscpu | grep "Model name"')
+    model_name = model_name.partition(":")[2]
+
+    print(">"*20,"Hardware Information",">"*20)
+    print('CPU architecture:',platform.processor())
+    print('CPU model name:',model_name.lstrip())
+    print('Logical Core(s):',psutil.cpu_count()) 
+    print('Physical Core(s):',psutil.cpu_count(logical=False)) 
+    print('Physical Core(s) per socket:',int(psutil.cpu_count(logical=False)/socket_num))
+    print('Socket(s):',socket_num)
+    print('CPU usage:',str(psutil.cpu_percent()) + '%')
+    print('CPU MHz:',format(psutil.cpu_freq().current,'.2f'))
+    print('CPU max MHz:',format(psutil.cpu_freq().max,'.2f'))
+    print('CPU min MHz:',format(psutil.cpu_freq().min,'.2f'))
+    print('Total memory:',get_bytesize(psutil.virtual_memory().total))
+    print('Available memory:',get_bytesize(psutil.virtual_memory().available))
+
+    #support instruction set or not
+    flag1 = int(subprocess.getoutput('cat /proc/cpuinfo | grep "flags"| sort -u | grep -c "avx512bw" '))
```

Comment:
change to check avx512f rather than avx512bw

Reply:
is this cmd useful here?
https://github.com/intel-analytics/BigDL/blob/e824af48695cd439be7b47e1932221b4d3d30788/python/nano/src/bigdl/nano/common/common.py#L31